lib/search/global/profiles.js

```diff
@@ -75,6 +75,15 @@ module.exports = client => async (term = '', query = {}) => {
           fuzziness: 'AUTO'
         }
       }
+    },
+    {
+      wildcard: {
+        email: {
+          value: `${term}*`,
+          boost: 5,
```

Comment:
I'm wary of boosting this so hard as it means that if someone's full name forms a part of someone else's email address then the partial email match will get boosted above the exact name match.

Can we leave it unboosted?